tests/v2/integration/steveapi/steve_api_test.go

```diff
@@ -238,14 +253,17 @@ func (s *SteveAPITestSuite) SetupSuite() {
 			Name:     username,
 			Enabled:  &userEnabled,
 		}
-		userObj, err := s.client.Management.User.Create(userObj)
+		userObj, err := users.CreateUserWithRole(client, userObj, "user")
```

Comment:
Would "user-base" be sufficient here, since we're adding cluster-member and additional roles? I'm noticing all the JSONs for users b and c now have a "createTypes" field which only happens when users have permission to create resources for that schema, which shouldn't be necessary for these tests.

Reply:
It could be when I was debugging the failures in the first place I thought this was the issue. I can change it back and rerun it.

Reply:
Changed it back